redwood-lazylayout-composeui/src/commonMain/kotlin/app/cash/redwood/lazylayout/composeui/ComposeUiLazyList.kt

```diff
@@ -102,9 +104,14 @@ internal class ComposeUiLazyList :
     this.margin = margin
   }
 
+  override fun crossAxisAlignment(crossAxisAlignment: CrossAxisAlignment) {
+    this.crossAxisAlignment = crossAxisAlignment
+  }
+
   override val value = @Composable {
     val content: LazyListScope.() -> Unit = {
       items(items.widgets) { item ->
+        // TODO If CrossAxisAlignment is Stretch, pass Modifier.fillParentMaxWidth() to child widget.
```

Comment:
Does not seem possible with the current design.

We could change the widget type for Compose UI from `@Composable () -> Unit` to `@Composable (Modifier) -> Unit` but that has huge rippling effects and I'm not actually sure it's even the right thing to do. It would mean that every Compose UI widget would need to accept a `Modifier` from its parent and attach it appropriately to itself.

Reply:
Yep, agreed that propagating a `Modifier` isn't the best solution. I'll just punt this to a future PR.